db/models.py

```diff
@@ -1,4 +1,9 @@
+from django.contrib.auth.models import AbstractUser
+from django.core.exceptions import ValidationError
 from django.db import models
+from django.db.models import CASCADE, UniqueConstraint
+
+import settings
```

Comment:
You are importing the settings module directly. It's a better practice to use Django's built-in functions to get settings values. For instance, you can use `from django.conf import settings` to get the settings values.